### PR TITLE
custom error policy tracers for shelley clnt/serv

### PR DESCRIPTION
**Pull request by avieth**
I don't know of a way to get a Text directly from an ErrorPolicyTrace,
so the Show instance is used

### CODE REVIEW

File: src/exec/Main.hs [hunk 1]
```diff
@@ -523,6 +526,20 @@ runByron tracer byronOptions genesisConfig blockConfig updateConfig nodeConfig e
     (\tbuilder -> (Text.pack "main.client", Monitoring.Info, tbuilder))
     tracer
 
+-- | For use by the Shelley server/client error policy tracers.
+defineSeverity :: ErrorPolicyTrace -> Monitoring.Severity
```
Comment (avieth) [inline src/exec/Main.hs:530]: Copied from a typeclass instance in `cardano-explorer` @coot

**Review by karknu — APPROVED** [review]
LGTM.

File: src/exec/Main.hs (hunk 1 above)
Comment (coot) [reply src/exec/Main.hs:530]: Which was copied from `cardano-node` - we should move the orphans to `ouroboros-neteork`.

**Pull request merged**